nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3732,6 +3732,21 @@ These properties determine the behavior of the internal NiFi predictive analytic
 
 |====
 
+[[runtime_monitoring_properties]]
+=== Runtime Monitoring Properties
+
+Long-Running Task Monitor periodically checks the NiFi processor executor threads and produces warning logs and bulletin messages for those that have been running for a longer period of time.
+It can be used to detect possibly stuck / hanging processor tasks.
+Please note the performance impact of the task monitor: it creates a thread dump for every run that may affect the normal flow execution.
+The Long-Running Task Monitor can be disabled via defining no values for its properties, and it is disabled by default.
```

Comment:
For clarity, is it worth noting that both of these properties must be specified in order to enable the task?

Reply:
Good idea, I added a note on it.